hoodie-client/src/main/java/com/uber/hoodie/config/HoodieCompactionConfig.java

```diff
@@ -93,6 +95,10 @@
     // 200GB of target IO per compaction
     public static final String DEFAULT_COMPACTION_STRATEGY = LogFileSizeBasedCompactionStrategy.class.getName();
 
+    // used to merge records written to log file
+    public static final String DEFAULT_MERGE_STRATEGY = HoodieAvroPayload.class.getName();
+    public static final String MERGE_STRATEGY_CLASS = "hoodie.compaction.merge.strategy";
```

Comment:
Currently merge strategy, payload is all the same.. So lets just call it "PAYLOAD_CLASS" instead of "MERGE_STRATEGY_CLASS", for the sake of not adding a new concept?

Reply:
Yeah, will make that change, after these changes I agree with not introducing a new concept.